utils/common.py

```diff
@@ -29,15 +30,18 @@ def extract_zip_to_temp_dir(application_path):
     yield tempdir.name
 
 def run_containerless_parametrize(func):
+    args_list = [{"--run-local=true": None}]  # Always include local mode
+
+    if platform.system().lower() != "windows":
+        args_list.append({"--run-local=false": None})  # Add container mode only if not Windows
```

Comment:
Is there a way we can make this more obvious, it could be confussing if we see a failure  with logs of containerized analysis in a test that has the word "containerless" on it.

Feels like we are creating a bit of knowledge niche that could be difficult for new contributors to understand.